docs/04-Getting-Started.md

```diff
@@ -253,6 +253,91 @@ services:
 
 Note that this Docker Compose file is untested. If this doesn't work for you, please open a pull request.
 
+#### Alternate docker-compose file
+This file HAS been tested (on a raspberry pi), but uses named volumes, which are not strictly neccesary.
+
+These settings are:
+- Named volumes (They may or may not need the driver and driver-opts sections, has not been tested without them. They definitely work with them in.)
+- **NB:** You must add the named volume folders to the same directory as your docker-compose file _before you run it_. Weird stuff happens if you don't.
+- Relative filepaths work fine with docker-compose, as long as they're relative to the location of the docker-compose file.
+- However, absolute filepaths are always the safest option.
+
+```yaml
+version: "3"
+
+services:
+  tribblewiki:
+    image: pepperminty-wiki
+    container_name: wiki
+    restart: unless-stopped
+    volumes: 
+      - wiki-app:/srv/app
+      - wiki-data:/srv/data
+
+volumes:
+  wiki-app:
+    name: "WikiAppFolder"
+    driver: local
+    driver_opts:
+      type: 'none'
+      o: 'bind'
+      device: './app'
+  wiki-data:
+    name: "WikiDataFolder"
+    driver: local
+    driver_opts:
+      type: 'none'
+      o: 'bind'
+      device: './data'
+```
+
+#### How to use the alternate docker-compose file.
+This guide is for Linux. Windows machines will need different commands etc.
+
+1. Build the container (see above) on the machine you want to run the container on.
+	- Building it on the machine itself means you don't need to worry about motherboard/kernal type, it will build the correct one for that machine.
+	- If you need to install Apache in order to install PHP so you can build it, remember to turn the 'start on boot' setting off. (Also just turn it off straight away, you don't need it to build.)
```

Comment:
The advice I got from a quick google is the easiest way to install php is to just install Apache? Haven't used PHP for years before I did this, so if there's an actual better way feel free to get rid of that line. (I did manage to confuse myself with it though so idk might be useful, but I THINK I also put it in the troubleshooting so that should be good enough.)